haskell/src/Stablecoin/Client/Parser.hs

```diff
@@ -168,9 +168,11 @@ data GetMintingAllowanceOptions = GetMintingAllowanceOptions
 
 data ContractMetadataOptions
   = OpCurrentContract -- ^ Indicate that we should store metadata in the main contract itself.
+      (Maybe Text) -- ^ An optional description of the contract to add to the contract's metadata.
   | OpRemoteContract -- ^ Indicate we should originate a dedicated contract to store metadata.
+      (Maybe Text) -- ^ An optional description of the contract to add to the contract's metadata.
```

Comment:
I suppose `add` is not quite correct here, because if non is specified the default one will be used?

Maybe replace it with:
```suggestion
      (Maybe Text) -- ^ An optional contract description to replace the default one in its metadata.
  | OpRemoteContract -- ^ Indicate we should originate a dedicated contract to store metadata.
      (Maybe Text) -- ^ An optional contract description to replace the default one in its metadata.
```

Reply:
I personally prefer to use "add" here. "Replace" implies something was already there, which you removed and replaced with something else.
In this case, I'm adding the given description to the contract's metadata, or, if none is given, I'm adding a default description.

I updated the comment to mention what happens when no description is given.

Reply:
Good point, sounds good to me.